feat(dynamodb): TableV2 supports imports, drift detection, and better control over replicas (#27023)
aws/aws-cdk: packages/aws-cdk-lib/aws-dynamodb/README.md

# Amazon DynamoDB Construct Library

> > The DynamoDB construct library has two table constructs - `Table` and `TableV2`. `TableV2` is the preferred construct to use for creating a single table or a table with multiple `replicas`.

Reviewer: Not blocking, but I'd probably phrase this even stronger. Something like
```suggestion
> The DynamoDB construct library has two table constructs - `Table` and `TableV2`. `TableV2` is the preferred construct for all use cases, including creating a single table and a table with multiple `replicas`.
```

Author: Updated.
